### PR TITLE
Add SymbolsToIndex to synonyms

### PR DESCRIPTION
**Pull request by Abfelbaum**
We need the `symbols_to_index` (can be found [here](https://typesense.org/docs/0.25.1/api/synonyms.html#create-or-update-a-synonym), last argument) on synonyms for our use case.

I added them to the `SynonymSchema` and `SynonymSchemaResponse` according to `SymbolsToIndex` from `Schema` and `CollectionResponse` and extended the existing tests to include the parameter. I'm not quite sure if that's correct tho.

I don't know why my IDE added the two using statements but when I remove them stuff won't build. I can remove them when they are not wanted.

### CODE REVIEW

File: test/Typesense.Tests/TypesenseClientTests.cs [hunk 1]
```diff
@@ -1,10 +1,10 @@
-using FluentAssertions;
-using FluentAssertions.Execution;
 using System;
 using System.Collections.Generic;
 using System.Linq;
 using System.Text.Json.Serialization;
 using System.Threading.Tasks;
+using FluentAssertions;
+using FluentAssertions.Execution;
 using Xunit;
```
Comment (runeanielsen) [inline test/Typesense.Tests/TypesenseClientTests.cs:8]: Can you undo the changes here so they're alphabetically sorted like before the changes?

**Review by runeanielsen — CHANGES_REQUESTED** [review]
Hi @Abfelbaum, thanks for making the contribution.

I've a small change that needs to be fixed. In the `TypesenseClientTests.cs` file the order of the usings were changes for no good reason. If you can change them back to the way they were ordered before I'll merge the changes in.


// Rune

File: test/Typesense.Tests/TypesenseClientTests.cs (hunk 1 above)
Comment (Abfelbaum) [reply test/Typesense.Tests/TypesenseClientTests.cs]: Ohh now I see what happened, the diff was weird before :D

I removed that from the commit.

**Review by runeanielsen — APPROVED** [review]

**Comment by runeanielsen** [comment]
Thanks for making the changes. It has now been merged in and I've made a new release https://github.com/DAXGRID/typesense-dotnet/releases/tag/6.13.0.
